ffmpeg-plugin/README.md

```diff
@@ -99,6 +99,13 @@ TBD
       -port 9001 -
    ```
 
+   When working with raw video files that lack metadata, you must explicitly provide ffmpeg with the necessary video frame details. This includes specifying the format `-f rawvideo`, pixel format `-pix_fmt` and resolution `-s WxH`. For example:
+
+    ```bash
+        ... ffmpeg -f rawvideo -pix_fmt yuv422p10le -s 1920x1080 -i <video-file-path> ...
```

Comment:
```suggestion
ffmpeg -f rawvideo -pix_fmt yuv422p10le -s 1920x1080 -i <video-file-path> ...
```

Reply:
I added "..." because in command above we suggest to use "sudo MCM_MEDIA_PROXY_PORT=8001 " before "ffmpeg"

Reply:
I understand. But are you sure the user does?
I suggest removing that.